config-connector/solutions/iam/kpt/kms-key-ring/README.md

```diff
@@ -0,0 +1,39 @@
+KMS Key Ring
+==================================================
+
+# NAME
+  kms-key-ring
+# SYNOPSIS
+  Config Connector compatible yaml files for applying IAM roles to a kms key ring
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/kms-key-ring kms-key-ring
+  ```
+# REQUIREMENTS
+  A working cluster with Config Connector installed managing a project with Cloud Key Management Service API enabled.
```

Comment:
A working cluster with Config Connector installed `and configured to manage` a project with 
`Cloud Key Management Service (KMS) API` enabled.

Let's be clear that it is Config Connector that has been configured and used to manage resources in projects. And also use the matched API name on Cloud Console.

Reply:
change made